packages/siwe/src/mapToSIWX.ts

```diff
@@ -96,18 +106,21 @@ export function mapToSIWX(siwe: AppKitSIWEClient): SIWXConfig {
     },
 
     async revokeSession(_chainId, _address) {
-      if (await siwe.signOut()) {
+      try {
+        await siwe.signOut()
         siwe.methods.onSignOut?.()
-
-        return Promise.resolve()
+      } catch (error) {
+        console.warn('AppKit:SIWE:revokeSession - signOut error', error)
       }
-
-      throw new Error('Failed to sign out')
     },
 
     async setSessions(sessions) {
       if (sessions.length === 0) {
-        await siwe.methods.signOut()
+        try {
+          await siwe.methods.signOut()
```

Comment:
how come one is siwe.signout and the other is siwe.methods.signout()

Reply:
it should be `siwe.methods`, good catch. This happens because of how [this class](https://github.com/reown-com/appkit/blob/main/packages/siwe/src/client.ts#L49) is built.